src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -229,7 +229,9 @@ where node.IsKind(SyntaxKind.IdentifierName)
             {
                 if (ctor.Initializer != null)
                 {
-                    bodyTokens = ctor.Initializer.DescendantTokens().Concat(bodyTokens);
+                    bodyTokens = bodyTokens != null
+                        ? ctor.Initializer.DescendantTokens().Concat(bodyTokens)
+                        : ctor.Initializer.DescendantTokens();
```

Comment:
❔ Is this better, or would it be better to modify the definition above to have this:

```
... ?? Enumerable.Empty<SyntaxToken>()
```